components/portfolio/PortfolioTechnologyTags.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const links = {
+  meteor: 'tech-back-end-meteor',
+  react: 'tech-front-end-react',
+  reactnative: 'tech-front-end-react-native',
+  mongo: 'tech-data-base-mongo',
+  express: 'tech-back-end-express',
+  redux: 'tech-front-end-redux',
+  nodejs: 'services-web-development-node',
+  cordova: 'tech-apps-cordova',
+  angular: 'tech-front-end-angular',
+  angular2: 'tech-front-end-angular-2',
+  android: 'services-mobile-development-android',
+};
+
+const PortfolioTechnologyTags = ({ tags, style }) => (
+  <div className="technology-tags-wrap">
+    <ul className="technology-tags--list">
+      {
+        tags.map((tag) => {
+          const href = links[tag.toLowerCase().replace(' ', '')];
+          return (
+            <li className="technology-tags--item" key={tag}>
+              {
+                href ? (
+                  <a
+                    className="technology-tag"
+                    href={`/${href}`}
```

Comment:
@soloviovmax , I'd rather put that slash into value of `links` object so that it actually contains links, not just link partials. But that's a minor thing, so feel free to disregard this if you don't have time to fix it.